src/components/shared/ButtonLibrary.tsx

```diff
@@ -75,5 +77,5 @@ export const OverlayButton: React.FC<OverlayButtonProps> = (props) => {
     if (style) {
         buttonStyle = { ...buttonStyle, ...style };
     }
-    return <TertiaryButton {...props} style={buttonStyle} />;
+    return <TertiaryButton size="large" {...props}  style={buttonStyle} />;
```

Comment:
nit: added stray whitespace here
```suggestion
    return <TertiaryButton size="large" {...props} style={buttonStyle} />;
```